### DISCUSSION TITLE
What's in a name? gotk is a mouthful!

### DISCUSSION THREAD

**Discussion by allymparker** [fluxcd/flux2#336] [Proposals] [answered]
So this is going to come across a bit silly, but...

I've been using the toolkit with my team for the last week or so, and something we've all struggled with is how to pronounce `gotk` when speaking to each other. 

"gee-oh-tee-kay" is a bit of a mouthful, and "goh-tuk" isn't much better...

From a usability perspective, I was wondering if a more friendly CLI name would help?

Some ideas below :)  

`gop`
`gops` 
`got` - Edit: Just had a thought - this could be said "go-tee" lol
`fluxctl`
`goctl` (A bit too similar to `golang` maybe)

Thoughts?

**Comment by stefanprodan** [comment, marked as answer]
Please see https://github.com/fluxcd/toolkit/discussions/367

**Reply by allymparker** [reply]
I still think go-tee would have been cool :P
